core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -408,6 +407,16 @@ export class JointUIService {
       .attr("rect/fill", JointUIService.getOperatorFillColor(operator));
   }
 
+  public changeOperatorDisplayName(
```

Comment:
maybe add a `joint` prefix?

Reply:
changed to `changeOperatorJointDisplayName`